### PR TITLE
[MIG] document_page_quality_manual: Migration to 14.0

### PR DESCRIPTION
**Pull request by dzungtran89**
Dependencies:
- document_page: https://github.com/OCA/knowledge/pull/283

### CODE REVIEW

File: test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+odoo14-addon-knowledge @git+https://github.com/dzungtran89/knowledge@14.0-mig-knowledge#subdirectory=setup/knowledge
+odoo14-addon-document-page @git+https://github.com/dzungtran89/knowledge@14.0-mig-document_page#subdirectory=setup/document_page
```
Comment (sebalix) [inline test-requirements.txt]: To clean up

**Review by sebalix — COMMENTED** [review]
LG, dependencies to clean up

File: test-requirements.txt (hunk 1 above)
Comment (dzungtran89) [reply test-requirements.txt]: @sebalix I updated it

**Review by simahawk — APPROVED** [review]

**Review by sebalix — APPROVED** [review]

**Pull request merged**